### DISCUSSION TITLE
How should libraries expose unstable features for those using nightly?

### DISCUSSION THREAD

**Discussion by gnzlbg** [rust-lang-nursery/api-guidelines#159] [General]
Should they use a feature flag, e.g., `unstable` ? Should they use a `--cfg` flag, e.g., `--cfg unstable` ? Should the flag be unique for each library, e.g., `--cfg rayon-unstable --cfg tokio-unstable` ?

**Comment by L-as** [comment]
Isn't it common to use a `nightly` feature?
